doc/source/tutorials/local/Adding_new_data/Select_write_store.rst

```diff
@@ -0,0 +1,286 @@
+Select a store to write to
+==========================
```

Comment:
Question: What steps does someone have to take if they want to add a second object store later on? Would they run quickstart again?

Reply:
Docs have now been updated.